test/integration/smartCache.js

```diff
@@ -0,0 +1,28 @@
+const app = require('../../server');
+const constants = require('../../app/lib/constants');
+const chai = require('chai');
```

Comment:
Conventionally, modules are included first (this applies to both `chai` and `chai-http`).